VictoriaAlbert/VictoriaAlbert/Records.swift

```diff
@@ -0,0 +1,78 @@
+//
+//  Records.swift
+//  VictoriaAlbert
+//
+//  Created by Cris on 11/10/16.
+//  Copyright © 2016 Cris. All rights reserved.
+//
+
+import Foundation
+
+enum RecordsModelParseError: Error {
+    case records
+}
+
+class Records {
+    let object: String
+    let date_text: String
+    let place: String
+    let image: String
+    let location: String
+    var smallImageURL: String {
```

Comment:
I like the way you implemented these in the model as computed properties. However, you hard coded that last segment to "2006AM" which is why some of your images are not loading.